meeko/macrocycle.py

```diff
@@ -231,15 +364,18 @@ def show_macrocycle_scores(self, setup):
             bond_by_ring = defaultdict(list)
```

Comment:
It doesn't look like this function is being used anywhere, so I have deprecated it for now. All this logic here is after a return statement and is therefore inaccessible code, so if we did want to keep this function we should give that some thought.